style.css

```diff
@@ -0,0 +1,128 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  margin: 0 auto;
+  height: 48px;
+  padding: 24px;
+}
+
+header .toolbar {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 20px;
+  color: #636fd4;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+p {
+  font-size: 4.3vw;
+}
+
+.my_logo {
+  margin-right: 200px;
+  margin-left: 10px;
+}
+
+#Introduction {
+  width: 100%;
+  height: calc(100vh - 48px);
+  background-image: url("images/header-shapes\ mobile.svg");
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  padding-top: 114px;
```

Comment:
- [ ] Kindly also consider lowering this padding to maybe `50px` because right now the gap between the navbar and hero section is bigger than the actual design. all that due to make our design almost identical to the figma design.
your design
![qw](https://user-images.githubusercontent.com/96683843/227016637-230213e8-cfaa-4b36-9852-52a7879772c9.jpg)

figma design
![as](https://user-images.githubusercontent.com/96683843/227016719-db2a4adc-052e-4545-880a-16bbbf6fb50a.jpg)